### PR TITLE
Add demo for training and serving with model-pool

### PR DESCRIPTION
**Pull request by zw0610**
This demo shows how to:

- [x] create a ModelVersion CR with PVC auto-generated
- [x] train a model and save the model to the storage provided by the PVC
- [x] serving with seldon with model stored in the PVC

@gaocegege @xieydd

### CODE REVIEW

**Comment by gaocegege** [comment]
What's the relationship between this PR and the PR in git.woa?

**Comment by zw0610** [comment]
@gaocegege For this repo `ml-demo`, there is no origin repository in our internal git. For the model-pool repository, it will remain internally before going public.

**Review by gaocegege — APPROVED** [review]
LGTM

File: mnist/tensorflow/with-model-pool/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Train and Serve a Model
+
+## Prerequisite
+
+Please make sure both SKAI/dashboard and SKAI/model-pool are installed.
```
Comment (xieydd) [inline mnist/tensorflow/with-model-pool/README.md]: Pls add install link for dashboard and model-pool.

File: mnist/tensorflow/with-model-pool/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+A PersistentVolumeClaim named `mlp-v1-pvc` will be created. User can probe the such information from the status of the modelversion.
+
+## Train
+
+Submit the [train.yaml](https://github.com/skai-x/ml-demo/blob/main/mnist/tensorflow/with-model-pool/train.yaml) to launch the training job.
+
+## Serving
+
+Submit the [serving.yaml](https://github.com/skai-x/ml-demo/blob/main/mnist/tensorflow/with-model-pool/serving.yaml) to launch the serving job.
```
Comment (xieydd) [inline mnist/tensorflow/with-model-pool/README.md:21]: How to verify the serving is running, and we real using the trained model.

**Review by xieydd — CHANGES_REQUESTED** [review]

File: mnist/tensorflow/with-model-pool/README.md (hunk 1 above)
Comment (zw0610) [reply mnist/tensorflow/with-model-pool/README.md]: fixed

File: mnist/tensorflow/with-model-pool/README.md (hunk 2 above)
Comment (zw0610) [reply mnist/tensorflow/with-model-pool/README.md:21]: https://github.com/skai-x/ml-demo/issues/3

**Comment by xieydd** [comment]
LGTM

**Review by xieydd — APPROVED** [review]

**Pull request merged**